sysutils/fatsort/Portfile

```diff
@@ -22,10 +22,10 @@ depends_build       port:help2man
 depends_lib         port:libiconv
 
 master_sites        sourceforge:${name}
-checksums           md5     f1232f40eba6ee9362acd9f0d5209dcf \
-                    sha1    9ae5d1ab5e4c91e6725237fab1271f881c3edb59 \
-                    rmd160  47e848fa3936d2bfc7f6e62f986055d312d809b9 \
-                    sha256  77acc374b189e80e3d75d3508f3c0ca559f8030f1c220f7cfde719a4adb03f3d
+distfiles           ${name}-${version}.tar.xz
```

Comment:
Do not set the `distfiles` directly. Just use the existing command for `xz`.

`use_xz             yes`